SPECS/kubernetes/kubernetes.spec

```diff
@@ -95,7 +95,8 @@ Pause component for Microsoft Kubernetes %{version}.
 %build
 # set version information using version file
 # (see k8s code: hack/lib/version.sh for more detail)
```

Comment:
This comment could use an update now, since we're not using the version file anymore and to explain what we're doing.